README.md

```diff
@@ -1 +1,126 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard# 
+This project is about simple  using webpack and gitflow.
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents <a name="readme-top"></a>
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+ 
+ This project is about simple  using webpack.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Html
+- CSS
+- Javascript ES6
+- Webpack
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://supreeti.github.io/Leaderboard/)
+  Visit my project.
```

Comment:
The live demo link is broken hence leads to a page with 404 error. Kindly ensure that your project is hosted online and link updated accordingly. You consider removing the whole live demo section if the app is not hosted yet to avoid leading to a broken page :wink: